.github/workflows/test-all.yml

```diff
@@ -154,6 +154,68 @@ jobs:
         github-token: ${{ secrets.GITHUB_TOKEN }}
         path-to-lcov: ./lcov-all.info
       continue-on-error: true
+  test-data-connect:
+    name: (Data Connect) Node.js and Browser (Chrome) Tests
+    needs: build
+    services:
+      postgres:
```

Comment:
Can you use the pglite that is built into the fdc emulator? I know it's a lot slower than a _real_ postgres database, but wanted to see if you considered that as an option and how you decided to use a real postres server.

Reply:
So, I don't think pglite comes with the pure binary, and because there isn't an existing infrastructure in this SDK to handle setting up and dealing with the CLI, I decided to go this route.

Reply:
There is an API call you can make to the emulator binary that sets up the schema and operations for you. 

https://github.com/firebase/data-connect-ios-sdk/blob/main/Tests/Integration/ConfigSetup.swift#L33

Reply:
Added